custom_reports/columns.rst

```diff
@@ -1,13 +1,15 @@
 Columns
 =======
 
-This article documents all available columns in the :ref:`custom-reports-http-api`.
+This article documents columns available in the :ref:`custom-reports-http-api`
+by default. Additional columns may become available through
```

Comment:
I would skip `by default` at the end of sentence. It isn't clear what it is referring to.
Alternatives:
```
This article documents columns available in the :ref:`custom-reports-http-api`.
```
or
```
This article documents core columns available in the :ref:`custom-reports-http-api`.
```